geos/cmake.win.patch

```diff
@@ -1,11 +1,11 @@
---- capi\CMakeLists.txt.orig	Wed Dec 10 18:54:08 2014
-+++ capi\CMakeLists.txt	Wed Dec 10 18:53:35 2014
+--- capi\CMakeLists.txt.orig	Sun Aug 25 15:10:32 2013
++++ capi\CMakeLists.txt	Mon Jan 19 15:10:24 2015
 @@ -25,7 +25,7 @@
-   # if building OS X framework, CAPI built into C++ library
-   add_library(geos_c SHARED ${geos_c_SOURCES})
-
+   # if building OS X framework, CAPI built into C++ library 
+   add_library(geos_c SHARED ${geos_c_SOURCES}) 
+ 
```

Comment:
Whitespace? Is this because of that bizarre issue that prevented the patch applying because the file to be patched itself has trailing whitespace in it?